alpha/declcfg/filter/filter.go

```diff
@@ -0,0 +1,15 @@
+package filter
+
+import (
+	"context"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+)
+
+type Filterer interface {
+	FilterCatalog(ctx context.Context, fbc *declcfg.DeclarativeConfig) (*declcfg.DeclarativeConfig, error)
+}
+
+type MetaFilterer interface {
+	KeepMeta(meta *declcfg.Meta) bool
+}
```

Comment:
Nit: Andy did a great job of getting this stuck in my head over the course of a few PRs: https://go.dev/wiki/CodeReviewComments#interfaces

The gist is that the interfaces should be defined where the values of the interface are used and not by the package that implements them. I'm not strongly opinionated on this (yet) but is something that crossed my mind while reviewing so figured I would point it out.

Reply:
Yep, this is an outcome of the way this code evolved before I submitted it. I'll move it to the `declcfg` package.